mrp_unbuild_cust_alu_analytics/views/mrp_unbuild_process_type_views.xml

```diff
@@ -14,6 +14,7 @@
                             </group>
                             <group>
                                 <field name="bom_line_ids" widget="many2many_tags"/>
+                                <field name="company_id" groups="base.group_multi_company" options="{'no_create': True}"/>
```

Comment:
Does it make sense that this field can be edited by the user?
```suggestion
                                <field name="company_id" groups="base.group_multi_company" readonly="1"/>
```

Reply:
It makes sense, see https://github.com/solvosci/slv-manufacture/pull/71#discussion_r1580746898